pkg/result/processors/nolint.go

```diff
@@ -232,9 +232,11 @@ func (p *Nolint) extractFileCommentsInlineRanges(fset *token.FileSet, comments .
 	return ret
 }
 
+var nolintRe = regexp.MustCompile(`^nolint( |:|$)`)
```

Comment:
can you move this variable to the top of the file?

Reply:
Sure.